libs/designer-ui/src/lib/dynamicallyaddedparameter/helper.ts

```diff
@@ -190,12 +230,26 @@ export function serialize(props: DynamicallyAddedParameterProps[]): ValueSegment
   ];
 }
 
-export function getEmptySchemaValueSegmentForInitialization(useStaticInputs: boolean) {
-  let rootObject: { type: string; properties: unknown; required: string[] } = {
-    type: 'object',
-    properties: {},
-    required: [],
-  };
+export function getEmptySchemaValueSegmentForInitialization(useStaticInputs: boolean, isManualTrigger = false) {
+  let rootObject;
+  if (isManualTrigger) {
+    rootObject = {
+      type: 'object',
+      properties: {},
+      required: [],
+    };
+  } else {
+    rootObject = {
```

Comment:
Curious, can you use the `default` property in manifest instead of using `getEmptySchemaValueSegmentForInitialization`?

Reply:
What would be the `default` in manifest? I don't see a default set

Reply:
I might be wrong here, I though manifest's capability to set a default value would help here but it may not be possible

Reply:
I might be wrong here, I though manifest's capability to set a default value would help here but it may not be possible